packages/web-components/src/components/cbp-nav-item/cbp-nav-item.scss

```diff
@@ -1,68 +1,79 @@
 /**
- * @Prop  --cbp-nav-item-button-color: var(--cbp-color-interactive-secondary-darker);
- * @Prop   --cbp-nav-item-button-color-dark: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop  --cbp-nav-item-color: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-color-dark: var(--cbp-color-interactive-secondary-lighter);
 
- * @Prop   --cbp-nav-item-button-color-hover: var(--cbp-color-interactive-secondary-darker);
- * @Prop   --cbp-nav-item-button-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-nav-item-color-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
 
- * @Prop   --cbp-nav-item-button-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
- * @Prop   --cbp-nav-item-button-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-nav-item-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
     
- * @Prop   --cbp-nav-item-button-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
- * @Prop   --cbp-nav-item-button-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-nav-item-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
 
  */
 
  :root {
-    --cbp-nav-item-button-color: var(--cbp-color-interactive-secondary-darker);
-    --cbp-nav-item-button-color-dark: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-nav-item-color: var(--cbp-color-interactive-secondary-darker);
+  --cbp-nav-item-color-dark: var(--cbp-color-interactive-secondary-lighter);
 
-    --cbp-nav-item-button-color-hover: var(--cbp-color-interactive-secondary-darker);
-    --cbp-nav-item-button-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-nav-item-color-hover: var(--cbp-color-interactive-secondary-darker);
+  --cbp-nav-item-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
 
-    --cbp-nav-item-button-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
-    --cbp-nav-item-button-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
-    
-    --cbp-nav-item-button-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
-    --cbp-nav-item-button-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-nav-item-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-nav-item-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
+  
+  --cbp-nav-item-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
+  --cbp-nav-item-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
 
```

Comment:
I don't see any styles for the current/selected item.

Reply:
updated styles to have the active state on the [selected] attribute